ds_judgements_public_ui/templates/judgment/detail.html

```diff
@@ -12,9 +12,9 @@
 {% endblock precontent %}
 {% block content %}
   <div id="start-of-document"></div>
-  {% if document %}
+  {% if document_html %}
```

Comment:
Would `document.get_content_as_html` do the trick?

Reply:
In the context it passes in:

```
MOST_RECENT_VERSION,
        query=preprocess_query(query) if query is not None else None,
```

Which I think if we added those to the context and passed them in in the template we've got the same issue.  I think having this one in the context is fine as it's not just an attribute on the document